install.sh

```diff
@@ -5,3 +5,6 @@
 git submodule init
 git submodule sync
 git submodule update
+
+# for react-native
+npm install react-native@0.27.2
```

Comment:
install.sh now needs to bring in the npm package for react native otherwise SalesforceReact won't compile